canals/pipeline/save_load.py

```diff
@@ -149,7 +138,13 @@ def unmarshal_pipelines(schema: Dict[str, Any]) -> Dict[str, Pipeline]:  # pylin
                     instance=component_instances[component_schema["refer_to"]],
                 )
             else:
-                component_class = classes[component_schema["type"]]
+                class_name = component_schema["type"]
+                if class_name not in component.registry:
+                    raise PipelineUnmarshalError(
+                        f"Failed loading Pipeline '{pipeline_name}'. Can't find Component class '{class_name}'. "
+                        "Please import that Component before loading."
```

Comment:
As Aga always said, don't use `Please`  in error messages :smile: 
```suggestion
                        "Make sure you imported this class before loading the pipelines."
```